CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## Unreleased
+
+**Breaking Changes**
+
+### Number[] Tokens [#1471](https://github.com/hashicorp/terraform-cdk/pull/1471)
+
+As part of an effort to use more native types, there are now tokens for number[].
+There is now `Token.asNumberList()` which can be used to conver other values into number[].
```

Comment:
I'd add that this is mostly used under the hood and end users probably don't have to care too much about it